{
  "file": "test/async/asyncNegative.js",
  "contents": "(The value of this property is over-sized, so it is validated independently.)",
  "attrs": {
    "description": "Async negative test",
    "negative": {
      "phase": "runtime",
      "type": "RangeError"
    },
    "flags": {
      "async": true
    },
    "includes": [
      "assert.js"
    ]
  },
  "copyright": "",
  "scenario": "default",
  "insertionIndex": 890,
  "source": "/*---\ndescription: Async negative test\nnegative:\n  phase: runtime\n  type: RangeError\nflags: [async]\n---*/\n\nsetTimeout(function() {\n  $DONE(new RangeError());\n}, 1000);\n"
}